types/share/share.go

```diff
@@ -2,9 +2,30 @@ package share
 
 import (
 	"github.com/celestiaorg/nmt"
+	"github.com/celestiaorg/rsmt2d"
+
 	"github.com/rollkit/celestia-openrpc/types/core"
 )
 
+var (
+	// DefaultRSMT2DCodec is the default codec creator used for data erasure.
+	DefaultRSMT2DCodec = rsmt2d.NewLeoRSCodec
+)
+
+const (
+	// NamespaveVersionSize is the size of a namespace version in bytes.
+	NamespaceVersionSize = 1
+
+	// NamespaceIDSize is the size of a namespace ID in bytes.
+	NamespaceIDSize = 28
+
+	// NamespaceSize is the size of a namespace (version + ID) in bytes.
+	NamespaceSize = NamespaceVersionSize + NamespaceIDSize
```

Comment:
`NamespaceVersionSize`, `NamespaceIDSize`, and `NamespaceSize` are also defined in `types/namespace/consts.go`. do they need to be redefined here?